src/main/kotlin/server/mcp/Task.kt

```diff
@@ -84,11 +84,20 @@ class Task(
             connection?.sendText(e.toString())
             connection?.sendText(stackDump())
         }
-        return Result.Finished
+        return Result.Finished(vReturn ?: VVoid)
     }
 
-    // Add a VM to the stack to run an exe.
-    fun push(
+    // Execute an exe immediately for a return value.  The task cannot suspend.
```

Comment:
Originally i was making a whole new Task to execute lambdas from the builtins, but I realized I could trick the existing Task into stopping at the depth of the new task and returning me the value.